cinema/views.py

```diff
@@ -0,0 +1,36 @@
+from django.shortcuts import render
+from rest_framework.generics import get_object_or_404
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        buses = Movie.objects.all()
+        serializer = MovieSerializer(buses, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    else:

```

Comment:
Explicitly check the action request method here and in the other places methods before performing the action.

Reply:
This was not resolved